src/markdown/templatemark@0.5.0.cto

```diff
@@ -84,10 +84,13 @@ concept OptionalDefinition extends BlockDefinition {
     o Child[] whenSome
     o Child[] whenNone
 }
-
 concept JoinDefinition extends BlockDefinition {
-    o String separator
-    o String lastSeparator optional
```

Comment:
Just double-checking that `lastSeparator` was intentionally removed here?

Reply:
Yes. Thanks!